server/src/apub/comment.rs

```diff
@@ -372,3 +388,54 @@ impl ApubLikeableType for Comment {
     Ok(())
   }
 }
+
+struct MentionsAndAddresses {
+  addressed_ccs: Vec<String>,
+  inboxes: Vec<String>,
+  tags: Vec<Mention>,
+}
+
+fn collect_non_local_mentions_and_addresses(
```

Comment:
Please add a comment explaining what this does. Also, what does `addresses` refer to?

Reply:
I added a description here now. Addresses are everything that goes into that `cc` field, so `community/followers`, and all the mentioned users.